tests/test_validation.py

```diff
@@ -717,10 +731,48 @@ def test_add_tokens_adapter(self):
             }
         )
 
-        validate_config(cfg)
+        check_model_config(cfg, model_config)
+
+    def test_phi2_add_tokens_adapter(self):
+        cfg = DictDefault(
+            {"adapter": "qlora", "load_in_4bit": True, "tokens": ["<|imstart|>"]}
+        )
+        model_config = DictDefault({"model_type": "phi-msft"})
```

Comment:
Would it be possible to load the actual config from their repo?

Reply:
for the model config? I suppose we could, but that would require fleshing out the `cfg` variable more. This is like the minimum required to test the function and is easy to grok.

Reply:
Ok, sounds good